fmriprep/cli/parser.py

```diff
@@ -222,11 +233,15 @@ def _slice_time_ref(value, parser):
     g_bids.add_argument(
         '-d',
         '--derivatives',
-        action='store',
-        metavar='PATH',
-        type=Path,
+        action=ToDict,
+        metavar='PACKAGE=PATH',
+        type=str,
         nargs='*',
```

Comment:
@effigies is there a reason `--derivatives` is set with `nargs='*'` instead of `nargs='+'`?